conf/msg_remove

```diff
@@ -1,15 +1,15 @@
 __APP__ was successfully removed :)
```

Comment:
This file isn't used anymore as the call to `ynh_send_readme_to_admin` was removed in the `remove` script ...

Imho this is fine to remove it because the only relevant message is about the fact that the data dir was not deleted, but anyway nowadays this should be handled by the core if I'm not mistaking ... (though it doesn't really explain how to remove the files for people that would have chosen not to purge data)